style.css

```diff
@@ -87,33 +116,35 @@
 
 .players-score-container {
   align-items: center;
-  border: solid purple 2px;
   display: flex;
-  height: 200px;
+  height: 225px;
   justify-content: space-evenly;
   width: 50%;
 }
 
 .player-one-container,
```

Comment:
We can probably add a second class to all of these within the HTML and reference it, instead of all 3. This will help to dry up some of the CSS